addons/mail/static/src/models/activity_view/activity_view.js

```diff
@@ -88,10 +165,28 @@ registerModel({
         areDetailsVisible: attr({
             default: false,
         }),
+        assignedUserText: attr({
+            compute: '_computeAssignedUserText',
+        }),
+        delayLabel: attr({
+            compute: '_computeDelayLabel',
+        }),
         fileUploaderView: one2one('FileUploaderView', {
             compute: '_computeFileUploaderView',
             inverse: 'activityView',
             isCausal: true,
         }),
+        formattedCreateDatetime: attr({
+            compute: '_computeFormattedCreateDatetime',
+        }),
+        formattedDeadlineDate: attr({
+            compute: '_computeFormattedDeadlineDate',
+        }),
+        markDoneText: attr({
+            compute: '_computeMarkDoneText',
+        }),
+        summary: attr({
+            compute: '_computeSummary',
+        }),
```

Comment:
Would be good to document each newly added field.
It makes it much easier to read and understand this model.